### DISCUSSION TITLE
[x][closed] Should we define two levels of the standard somehow?

### DISCUSSION THREAD

**Discussion by sveinugu** [fairtracks/fairtracks_standard#106] [Team Posts]
Some metadata can be autogenerated from others:

- Ontology URI -> Human readable version
- Set of ontology URI -> structured attributes, e.g. tissue type, cell type

Should we define two levels of the standard:
- Minimal set for data providers
- More expansive set for tools/script users

**Comment by dzerbino** [comment]
The second set seems like 'nice to have', i.e. they are in the spec, but not required. Defining an intermediary (not required, but recommended) strikes me a splitting hairs for the readers.

**Comment by sveinugu** [comment]
Yes. This was an old question that I copied without really thinking. I don't think we need this. We plan to add support for such autogeneration in TrackFind. If such autogenerated content should be available elsewhere (meaning TrackHub registry), we need to figure out a pipeline to get curated content piped back from TrackFind into the TrackHub registry. But that is for later. Closing this now

**Comment by sveinugu** [comment]
Discussion continued in https://github.com/orgs/fairtracks/teams/fairtracks-developers/discussions/46
